pom.xml

```diff
@@ -277,6 +278,12 @@
             <groupId>org.apache.spark</groupId>
             <artifactId>spark-sql_${scala.major.version}</artifactId>
             <version>${spark.version}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>${slf4j-api.version}</version>
 			<scope>provided</scope>
```

Comment:
I see that the scope is set to 'provided'. Does spark has a dependency on slf4j-api? Are we sure if it will be available in runtime or should user include it as a dependency to their project? Could you please check this. Please also check that spark-shell is not broken.

Reply:
Yes, I've checked. The logging libraries were in spark-core